sdk/operationalinsights/ci.yml

```diff
@@ -1,30 +1,43 @@
 # DO NOT EDIT THIS FILE
 # This file is generated automatically and any changes will be lost.
 
+resources:
+  repositories:
+    - repository: azure-sdk-tools
+      type: github
+      name: Azure/azure-sdk-tools
+      endpoint: azure
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+
 trigger:
   branches:
     include:
     - master
+    - feature/*
+    - hotfix/*
+    - release/*
   paths:
     include:
-    - sdk/operationalinsights/
+    - sdk/operationalinsight/
```

Comment:
This trigger path looks like it should have an `s` based on the directory this file lives in.

Reply:
Similar mistake in other places in this file.